quodlibet/docs/development/plugins.rst

```diff
@@ -5,7 +5,7 @@ Plugin Development
 
 A Quod Libet / Ex Falso Plugin is a simple python module or package which 
 contains sub-classes of special plugin classes and is placed anywhere in 
-``~/.quodlibet/plugins`` so it can be found by QL. These classes can provide 
+``~/.config/quodlibet/plugins`` so it can be found by QL. These classes can provide 
```

Comment:
Thanks, this is good. The true location is a bit more complex for this, but this is at least more correct than what was there.